src/components/Editor/GeneralInformation/BaseFields.vue

```diff
@@ -364,16 +365,25 @@
                 "icons",
                 "status"
             ]),
+            ...mapState('users', ['user']),
             fields(){
               return this.getSection("generalInformation").data;
             }
         },
         methods: {
-            removeCountry(country){
-                this.fields.countries = this.fields.countries.filter(obj =>
-                    obj.label !== country.name && obj.id !== country.id
-                );
+          removeCountry(country){
+              this.fields.countries = this.fields.countries.filter(obj =>
+                  obj.label !== country.name && obj.id !== country.id
+              );
+          },
+          typeChangeDisabled(){
+            let _module = this;
+            if (_module.getCreatingNewRecord) {
+              return false;
+            } else {
+              return !_module.user().is_curator;
             }
```

Comment:
else is not required:
```suggestion
            if (_module.getCreatingNewRecord) {
              return false;
            } 
            return !_module.user().is_curator;
```